packages/jamf_protect/data_stream/alerts/sample_event.json

```
@@ -1,24 +1,24 @@
 {
-    "@timestamp": "2024-09-01T20:43:13.311Z",
+    "@timestamp": "2024-09-03T08:14:23.798Z",
     "agent": {
-        "ephemeral_id": "17fd4566-0d6d-4a3b-870d-2b55cb00219b",
-        "id": "eb9762c2-ef0d-4e97-8e71-b8f31265d39c",
+        "ephemeral_id": "0b7b5d6b-ec4c-4c9f-879f-6d07308121e9",
+        "id": "e176809c-0a61-421a-a0e2-e1c422c3d93f",
         "name": "docker-fleet-agent",
         "type": "filebeat",
-        "version": "8.13.0"
+        "version": "8.13.2"
```

Let's keep the version here matching what the manifest claims.
```suggestion
        "version": "8.13.0"
```